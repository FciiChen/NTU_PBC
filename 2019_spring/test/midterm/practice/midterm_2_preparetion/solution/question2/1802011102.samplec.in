John , M ,   35 , Student
Mary , F ,  45    , Teacher
Bob,  M ,  12 ,  Cook
Kobe, M   , 40  , Cook
BREAK
MARY, 22
Bob, 75
BREAK