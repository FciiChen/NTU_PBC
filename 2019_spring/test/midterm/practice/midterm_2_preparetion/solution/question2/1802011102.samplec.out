John       M          35         Student    NULL      
Mary       F          45         Teacher    22        
Bob        M          12         Cook       75        
Kobe       M          40         Cook       NULL      
